index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Startup Matchmaker</title>
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+  </head>
+
+  <body>
+    <div class="container">
+
+      <header>
+        <h1>Startup Matchmaker</h1>
+        <nav>
+          <a href="#" class="caps">Find Developers</a>
+          <a href="#" class="caps">Find Designers</a>
+          <a href="#">How it Works</a>
+          <a href="#">Our Team</a>
+          <a href="#">Blog</a>
+        </nav>
+      </header>
+      
+      <hr>
+
+      <div class="tagline">
+        <h3>Because two brains <strike>taste</strike> <span>are</span> better than one.</h3>    
```

Comment:
Try to avoid using decorative tags for styling, defering to css gives you more control and flexibility.  To achieve this in css you would use `text-decoration: line-through`.